include/sys/fs/zfs.h

```diff
@@ -908,6 +909,9 @@ typedef enum pool_scan_func {
 typedef enum pool_scrub_cmd {
 	POOL_SCRUB_NORMAL = 0,
 	POOL_SCRUB_PAUSE,
+	POOL_RESCRUB_NORMAL,
+	POOL_RESCRUB_PAUSE,
+	POOL_RESCRUB_STOP,
```

Comment:
Rather than introducing new `pool_scrub_cmd_t`'s could we instead pass a flag in the ioctl's `innvl` requesting a "rescrub" instead of a normal scrub.

Reply:
Removed `POOL_RESCRUB_NORMAL` and `POOL_RESCRUB_PAUSE` as `POOL_SCRUB_NORMAL` and `POOL_SCRUB_PAUSE` can be used along with rescrub command pool_scan_func. But `POOL_RESCRUB_STOP` is cant be removed. `POOL_RESCRUB_STOP` is required to stop error scrub not normal scrub.